homeassistant/components/tado/tado_connector.py

```diff
@@ -324,7 +324,7 @@ def set_meter_reading(self, reading: int) -> dict[str, Any]:
         """Send meter reading to Tado."""
         dt: str = datetime.now().strftime("%Y-%m-%d")
         if self.tado is None:
-            raise HomeAssistantError("Tado client is not initialized")
+            raise ServiceValidationError("Tado client is not initialized")
```

Comment:
I think you addressed the wrong comment...

Reply:
Whoops, sloppy. Sorry. Fixed!